src/multislice.py

```diff
@@ -0,0 +1,124 @@
+from simSPI import fourier
+import numpy as np
+
+def apply_complex_ctf_to_exit_wave(exit_wave_f,complex_ctf):
+    """Apply complex contrast transfer function to multisclice exit wave.
+
+    Fourier based convolution of exit wave and ctf, and collapse of wave function.
+
+    Parameters
+    ----------
+    exit_wave_f : numpy.ndarray, shape (N,N)
+        exit wave array.
```

Comment:
NIT: Uppercase letter on the description of the parameters: we are just enforcing this so that all the docstrings parameters descriptions have the same convention, because that's how the (future) doc website will look clean too.

Reply:
i.e. `Exit wave array.`

Reply:
ok!